ovos_utils/version.py

```diff
@@ -1,8 +1,7 @@
 # The following lines are replaced during the release process.
 # START_VERSION_BLOCK
 VERSION_MAJOR = 0
-VERSION_MINOR = 0
-VERSION_BUILD = 38
-VERSION_POST = 0
+VERSION_MINOR = 2
```

Comment:
Should be 0.1.0; is there a bug in the version automation?

Reply:
seems like a bug, im not familiar with the current automations, think its the first time i tried a "propose" action

possible i misclicked?

Reply:
Might be related to the automation too; I generally leave the `alpha` track on an incremented "BUILD" version, rather than the next "MINOR".

Going forward, how do we want to deal with versioning?

- PR to `dev` is responsible for specifying major/minor version changes. This means we might go from 0.1.1a1 to 0.2.0a1 if a feature makes it in after a bugfix
- alpha track always reflects the next patch version and PR to `master` specifies the semver change. This means in this case we would have had 0.0.39aX that would now be releasing as 0.1.0
- Something else?

Reply:
we are supposed to move to the new automations  https://github.com/OpenVoiceOS/.github/tree/feat/shared_actions1

per https://openvoiceos.github.io/ovos-technical-manual/versioning/